linkerd/http-metrics/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "linkerd2-http-metrics"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+bytes = "0.4"
+futures = "0.1"
+h2 = "0.1"
+http = "0.1"
+http-body = "0.1"
+httparse = "1.2"
+hyper = "0.12.3"
+indexmap = "1.0"
+linkerd2-error = { path  = "../error" }
+linkerd2-http-classify = { path  = "../http-classify" }
+linkerd2-metrics = { path  = "../metrics" }
+rand = "0.7"
```

Comment:
what do we use `rand` for in this crate? i didn't see it in the code.

Reply:
in general, i think we may want to run whatever the cargo tool that tells you if dependencies are actually used?

Reply:
For what it's worth, I did run the tool and it didn't catch any of this :man_shrugging: